samples/nrf5340/empty_net_core/prj.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Intentionally left empty
```

Comment:
Don't see a point in this file if it's going to be empty

Reply:
Are there actually any samples or applications that do not provide the `prj.conf` file? I noticed that `prj.conf` is not mandatory (we can build a sample without it), but it seems that each application defines it (e.g, Zephyr's `hello_world` sample and `application_development/out_of_tree_board` samples define an empty `prj.conf` file). Shouldn't we leave the conf file for consistency with the common convention?

Reply:
There are quite a few that do not have it, things like hello_world have it because it's a fundamental sample that many people copy for the base of their application.

Reply:
Removed.